src/components/app/details/triggerView/workflow/Workflow.tsx

```diff
@@ -252,6 +256,10 @@ export class Workflow extends Component<WorkflowProps> {
         })
     }
 
+    handleWorkflowSelection() {
```

Comment:
we can directly use arrow function instead of binding

Reply:
done